### PR TITLE
bump up org.seleniumhq.selenium to 2.48.2

### PR DESCRIPTION
**Pull request by prabhjyotsingh**
With current version i.e. 2.44.0 latest version of firefox is not supported, hence upgrading it to 2.48.2

### CODE REVIEW

File: cassandra/pom.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
         <lz4.version>1.2.0</lz4.version>
         <scala.version>2.11.7</scala.version>
         <scala.binary.version>2.11</scala.binary.version>
-        <commons-lang.version>3.3.2</commons-lang.version>
+        <commons-lang.version>3.4</commons-lang.version>
```
Comment (felixcheung) [inline cassandra/pom.xml]: is this intentional? this is not mentioned in the PR description

Comment (prabhjyotsingh) [inline cassandra/pom.xml]: Yes, org.seleniumhq.selenium - 2.48.2 requires commons-exec:commons-exec - 1.3 which in turn will require commons-lang3 - 3.4. Hence, had to upgrade all.

**Comment by Leemoonsoo** [comment]
Thanks for bumping up the version. Finding old version of firefox was pain.
One small thing is, could you also update the comment about firefox version requirement here https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-server/src/test/java/org/apache/zeppelin/ZeppelinIT.java#L47 ?

**Comment by prabhjyotsingh** [comment]
@Leemoonsoo sure, thanks a lot. Totally missed it.

**Comment by prabhjyotsingh** [comment]
Can we merge this, if there is discussion around this ?

**Comment by Leemoonsoo** [comment]
I'm merging it!

**Pull request closed without merging**